yeelight/main.py

```diff
@@ -102,6 +124,24 @@ def last_properties(self):
         """
         return self._last_properties
 
+    @property
+    def bulb_type(self):
+        """
+        Returns BulbType :py:enum:`BulbType <yeelight.BulbType>` describing type
+        of the bulb. Currently this is Color or White.
+
+        When trying to access before properties are known, the bulb type is unknown.
+
+        :rtype: BulbType
+        :return: The bulb's type.
+        """
+        if not self._last_properties:
+            return BulbType.Unknown
+        if not any([self.last_properties[name] for name in ['ct', 'rgb', 'hue', 'sat']]):
```

Comment:
Was thinking about it but decided that it may be better to fallback to white if one of those fail. I'll consider it though, will need to do some retesting with the old bulb.